docs/zh/introduction/distinctive-features.md

```diff
@@ -37,7 +37,7 @@ ClickHouse会使用服务器上一切可用的资源，从而以最自然的方
 
 ## 支持SQL {#zhi-chi-sql}
 
-ClickHouse支持一种[基于SQL的声明式查询语言](../sql-reference/index.md)，它在许多情况下与[ANSI SQL标准](../sql-reference/ansi.md)相同。
+ClickHouse支持一种基于SQL的声明式查询语言。
```

Comment:
I have zero idea about Chinese and Russian. I trust you with the Russian version, will ask somebody to check this in any case.

Reply:
No need to bother, it looks ok in Google Translate.